Assignment4_FeatureEngineering.py

```diff
@@ -0,0 +1,260 @@
+import os
+import sys
+
+import matplotlib.pyplot as plt
+import numpy
+import pandas as pd
+import statsmodels
+import statsmodels.api
+from plotly import express as px
+from plotly import figure_factory, graph_objects
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+from sklearn.metrics import confusion_matrix
+
+
+def main(file, response):
```

Comment:
When i am trying to run it with data file I am getting error.
KeyError: "['median_house_value'] not found in axis"

Reply:
You renamed the target variable[suppose diagnosis ] to response..and to drop it you using the variable name.(which is diagnosis).it should be "response" ..

Reply:
This comment was so helpful. I was totally confused about it. 
Now I think it works fine.